metadata-ingestion/sink_docs/datahub.md

```diff
@@ -0,0 +1,75 @@
+# DataHub
+
+## DataHub Rest
+
+### Setup
+
+To install this plugin, run `pip install 'acryl-datahub[datahub-rest]'`.
+
+### Capabilities
+
+Pushes metadata to DataHub using the GMA rest API. The advantage of the rest-based interface
+is that any errors can immediately be reported.
+
+### Quickstart recipe
+
+Check out the following recipe to get started with ingestion! See [below](#config-details) for full configuration options.
+
+```yml
+sink:
+  type: "datahub-rest"
+  config:
+    server: "http://localhost:8080"
+```
+
+### Config details
+
+Note that a `.` is used to denote nested fields in the YAML recipe.
+
+| Field    | Required | Default | Description                  |
+| -------- | -------- | ------- | ---------------------------- |
+| `server` |          |         | URL of DataHub GMS endpoint. |
+
+## DataHub Kafka
+
+### Setup
+
+To install this plugin, run `pip install 'acryl-datahub[datahub-kafka]'`.
+
+### Capabilities
+
+Pushes metadata to DataHub by publishing messages to Kafka. The advantage of the Kafka-based
+interface is that it's asynchronous and can handle higher throughput. This requires the
+DataHub mce-consumer container to be running.
```

Comment:
Quick note: Now that GMS ships with MCE and MAE consumers embedded, we may not want to mention this! (it can be confusing)